editor/src/messages/portfolio/document/node_graph/node_properties.rs

```diff
@@ -1513,6 +1570,52 @@ pub(crate) fn _gpu_map_properties(document_node: &DocumentNode, node_id: NodeId,
 	vec![LayoutGroup::Row { widgets: map }]
 }
 
+pub(crate) fn grid_properties(node_id: NodeId, context: &mut NodePropertiesContext) -> Vec<LayoutGroup> {
+	let grid_type_index = 1;
+	let spacing_index = 2;
+	let angles_index = 3;
+	let rows_index = 4;
+	let columns_index = 5;
```

Comment:
Why hardcode the index instead of using the fun new system?

Reply:
For example `generator_nodes::isometric_grid::RowsInput::INDEX`.

Reply:
I wasn't aware of that system, but that's awesome to hear it exists now! There are plenty of nodes using indexes directly so this should be a refactor PR that changes them all over. Thanks for making me aware of it now.